middlewares/search.middleware.js

```diff
@@ -19,18 +19,44 @@ function parseQuery(req, res, next){
 
     req.body.q = JSON.parse(query);
 
+    //Default page
+    if(!req.body.hasOwnProperty("page")){
+        req.body.page = 0;
+    }
+    else{
+        req.body.page = parseInt(req.body.page);
+    }
+    //Default limit
+    if(!req.body.hasOwnProperty("limit")){
+        req.body.limit = 25;
+    }
+    else{
+        req.body.limit = parseInt(req.body.limit);
+    }
+    //Default sorting
+    if(!req.body.hasOwnProperty("sort")){
+        req.body.sort = '';
+        req.body.sort_by = '';
```

Comment:
is there a reason here that this is not camelCase?

Reply:
sort_by I think is the standard in queries passed through the url

Reply:
gotchu